packages/provider/src/utils.ts

```diff
@@ -34,6 +34,11 @@ export function sortConfig(config: ArcadeumWalletConfig): ArcadeumWalletConfig {
   return config
 }
 
+export function isUsableConfig(config: ArcadeumWalletConfig): boolean {
```

Comment:
what does the term "usable" mean? perhaps you could leave a comment above the function line to describe its use and scenario?

Reply:
Non-usable means that's valid but the sum of the signers is below the required threshold, I wanted to avoid the 'invalid' word, because the configuration is "valid" in some way.

I can add a comment, or we can change the nomenclature and consider configurations that don't reach the threshold to be "invalid"

Reply:
I think the "usable" sounds good -- just leaving a comment above as you mentioned is perfect. Keep it simple, in Go, a lot of people just leave a single comment in form or `// <function> ...` for example, `// isUsableConfig checks if a wallet configuration meets the necessary threshold to perform successful signing / transactions. A usable config can perform signing, and an unusable config means the user requires to add another signer to meet minimum security requirements.`